website/pages/five_safes_tes/quickstart/troubleshooting.mdx

```diff
@@ -0,0 +1,43 @@
+import { Steps, Cards, Callout } from "nextra/components";
+
+## Troubleshooting
+
+### 1. [Keycloak console](http://localhost:8085) requires HTTPS
```

Comment:
No need to add a number in front of the headings, if they're not step by step instructions